pkg/controller/wip/controller.go

```diff
@@ -153,7 +153,10 @@ func (c *controller) reconcileBroker(broker *v1alpha1.Broker) {
 	username, password, err := GetAuthCredentialsFromBroker(c.kubeClient, broker)
 	if err != nil {
 		glog.Errorf("Error getting broker auth credentials for broker %v: %v", broker.Name, err)
-		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason, errorFetchingCatalogMessage)
+		err := c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason, errorFetchingCatalogMessage)
+		if err != nil {
+			glog.Errorf("Error updating ready condition for Broker %v: %v", broker.Name, err)
+		}
```

Comment:
I'm not usually one to propose follow ups, but if the behavior is changed then the function probably no longer needs to return an error at all. I merely copied what is already present a few lines below.